### DISCUSSION TITLE
[http-server] has wrong type for createServer

### DISCUSSION THREAD

**Discussion by felix9** [DefinitelyTyped/DefinitelyTyped#59921] [Issues with a @types package]
`@types/http-server` says that `createServer` returns `http.Server | https.Server`, but it doesn't.

It actually returns an `HttpServer` that's internal to `http-server`, which only has a few methods that resemble `http.Server`, and there's a `.server` property that's the actual `http.Server` object.

[a comment by an automated account is not shown]

**Comment by plantain-00** [comment]
You can fix it by creating a PR.
